### PR TITLE
Modified graph node to cater for drawables

### PR DESCRIPTION
**Pull request by nikshasimney**
As per PR title :)

### CODE REVIEW

**Review by DuartBreedt — APPROVED** [review]
Very nice!

File: radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt [hunk 1]
```diff
@@ -98,12 +108,17 @@ class RadialGraph : ConstraintLayout {
...
+                Log.w(
+                    TAG,
+                    "No value passed for the `app:graphNodeColor` attribute. It will default to Magenta"
+                )
             }
 
             ContextCompat.getColor(context, R.color.node_defaultColor)
         }
 
+        val graphNodeIcon = attributes.getDrawable(R.styleable.RadialGraph_graphNodeIcon)
```
Comment (abdul-sayed22) [inline radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt:115]: Can we add a warning similar to the one above to check if the NodeType is set to `ICON` and no drawable was passed?

**Review by abdul-sayed22 — APPROVED** [review]

File: radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt (hunk 1 above)
Comment (nikshasimney) [reply radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt:115]: Should we perhaps provide a default icon as well?

File: example/src/main/res/drawable/ic_rewards_nav_icon.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ceNobiteElf) [inline example/src/main/res/drawable/ic_rewards_nav_icon.xml]: Should this icon be committed? As a visual asset, there are copyright issues with releasing this into the public domain if we do not have the explicit permission to do so.

File: radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt (hunk 1 above)
Comment (DuartBreedt) [reply radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt:115]: If a icon is not provided I think an error should be thrown.

File: example/src/main/res/drawable/ic_rewards_nav_icon.xml (hunk 2 above)
Comment (DuartBreedt) [reply example/src/main/res/drawable/ic_rewards_nav_icon.xml]: Rather use a material icon from Android studio. You can find them in the Vector Asset importer under clipart:
image

File: radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt [hunk 3]
```diff
@@ -221,11 +244,15 @@ class RadialGraph : ConstraintLayout {
 
                 val sectionNormalizedSize: BigDecimal = section.normalizedValue
 
-                val labelPositionValue: Float = calculateLabelPositionValue(section, labelStartPositionValue)
+                val labelPositionValue: Float =
+                    calculateLabelPositionValue(section, labelStartPositionValue)
```
Comment (ceNobiteElf) [inline radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt]: Not my repo, but the column width seems fairly low?

Comment (nikshasimney) [reply radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt]: @DuartBreedt should this be adjusted ?

Comment (DuartBreedt) [reply radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt]: Please 🤗

**Pull request merged**